manifests/repo/apt.pp

```diff
@@ -77,7 +77,12 @@
             /i\d86/      => 'i386',
             default      => false,
           }
-          $repo_url  = "http://download.gluster.org/pub/gluster/glusterfs/${release}/LATEST/Debian/${::lsbdistcodename}/apt/"
+          if versioncmp("${release}", '3.12') < 0 { 
+            $repo_url  = "http://download.gluster.org/pub/gluster/glusterfs/${release}/LATEST/Debian/${::lsbdistcodename}/apt/"
+          }
```

Comment:
We typically put `}` and `else` on the same line.

Reply:
Ok, I'll change this